File: Basic/Check Whether a Character is Vowel or Consonant/SolutionByGurirath.cpp
#include <iostream>
using namespace std;
int main()
{
    char ch;
    cout << "Enter the character" << endl;
    cin >> ch;
    if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' || ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U') {
        cout << "Vowel" << endl;
    } else {
        cout << "Consonant" << endl;
    }
    return 0;
}

File: Basic/Check Whether a Character is Vowel or Consonant/SolutionByGurirath.java
import java.util.*;
public class MyClass {
    public static void main(String args[]) {
        Scanner kb=new Scanner(System.in);
        char ch;
        System.out.println("Enter a character to check whether it is a consonant or a vowel");
        ch=kb.next().charAt(0);
        String n;
        n="AEIOUaeiou";
        if(n.indexOf(ch)==-1)
        {
            System.out.println("It is a consonant");
        }
        else
        {
            System.out.println("It is a vowel");
        }


    }
}
